src/particles/ImpactXParticleContainer.cpp

```diff
@@ -142,6 +135,7 @@ namespace impactx
          */
         auto old_np = particle_tile.numParticles();
         auto new_np = old_np + pinned_tile.numParticles();
+        amrex::AllPrint() << "old_np=" << old_np << " new_np=" << new_np << "\n";
```

Comment:
Needs to be updated in AMReX pls :)
See: https://github.com/AMReX-Codes/amrex/pull/2878#issuecomment-1261115487